src/test/kotlin/com/wgslfuzz/core/ParseTintTests.kt

```diff
@@ -93,7 +93,12 @@ class ParseTintTests {
                 text.contains("enable subgroups") ||
                 // binding_array is not currently mentioned in the WGSL specification, but online
                 // research suggests that it is a naga extension.
-                text.contains("binding_array")
+                text.contains("binding_array") ||
```

Comment:
Can you open an issue for @Max-Sepp to look into this, and replace the TODO with that issue?